test/framework/deployments/postgres/universal.go

```diff
@@ -0,0 +1,103 @@
+package postgres
+
+import (
+	"bytes"
+	"fmt"
+	"os/exec"
+	"regexp"
+	"strconv"
+
+	"github.com/gruntwork-io/terratest/modules/retry"
+	"github.com/gruntwork-io/terratest/modules/testing"
+
+	. "github.com/kumahq/kuma/test/framework"
+	. "github.com/kumahq/kuma/test/framework/deployments"
+)
+
+type universalDeployment struct {
+	name      string
+	container *DockerContainer
+	envVars   map[string]string
+}
+
+func (u *universalDeployment) GetEnvVars() map[string]string {
+	return u.envVars
```

Comment:
I still feel that it would be better to return some `PostgresConfiguration` object instead of map of Kuma env vars. I understand though that this might be hard because of cyclic dependencies? So I'm fine with this one.

Reply:
It's exactly because of cyclic dependencies